cli/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "cargo-chipmunk"
+version = "0.1.0"
+edition = "2021"
+description = "CLI Tool for chipmunk application development"
+
+[workspace]
+members = ["dir_checksum"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[workspace.dependencies]
+tempdir = "0.3"
+anyhow = "1.0.80"
+
+[dependencies]
+anyhow.workspace = true
+clap = { version = "4.4.4", features = ["derive"] }
+console = "0.15.7"
+fs_extra = "1.3.0"
+futures = "0.3.28"
+futures-lite = "1.13.0"
+git2 = "0.18.2"
+indicatif = "0.17.7"
+tokio = { version = "1.36.0", features = ["full"] }
+dir_checksum = { path = "./dir_checksum" }
+which = "6.0"
+clap_complete = "4.5"
```

Comment:
looks good, just make sure that for stable crates we leave out the bugfix part of the version.
tokio: do we need full here?